app/models/meeting.rb

```diff
@@ -0,0 +1,22 @@
+class Meeting < ActiveRecord::Base
+  has_many :meeting_attendances
+  has_many :attendances, through: :meeting_attendances, source: :user
+
+  validates :name, presence: true
+  validates :start_at, presence: true
+  validates :end_at, presence: true
+  validate :validate_datetime
+
+  def self.current
+    self.where('start_at < ? AND end_at > ?', Time.now, Time.now).first
+  end
+
+  private
+
+  def validate_datetime
+    return if start_at.blank? || end_at.blank?
+    if start_at > end_at
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.<br>Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.